WordPress/Classes/System/WordPressAppDelegate+openURL.swift

```diff
@@ -20,6 +20,19 @@ import AutomatticTracks
             return true
         }
 
+        /// handle the migration-related deeplink intents.
+        /// `UniversalLinkRouter` requires that it's in a universal link format (https://wordpress.com/<path>),
+        /// so we'll hackily converts the deeplink into an acceptable form.
+        if url.scheme == AppScheme.wordpressMigrationV1.rawValue,
```

Comment:
@twstokes I'll need to add the `UIApplication+Availability` here to check the scheme before executing the export on the WP side. Do you have any concerns about making `AppScheme` available to WP?

Reply:
Done ✅